src/gateway/understanding-kong/glossary.md

```diff
@@ -3,4 +3,36 @@ title: Glossary
 
 ---
 
-## PLACEHOLDER Glossary
+| Term      | Definition |
+| ----------- | ----------- |
+|Admin | An Admin is a {{site.base_gateway}} user account capable of accessing the Admin API or Kong Manager. |
+|Authentication| Authentication is the process by which a system validates the identity of a user account. |
+|Authorization| Authorization is the system of defining access to certain resources. In {{site.base_gateway}}, Role-Based Access Control (RBAC) is the main authorization mode. |
+|Beta| A Beta designation in Kong software means the functionality of a feature or release version is of high quality and can be deployed in a non-production environment. |
```

Comment:
For beta, tech preview/alpha, and GA, might be better to link off to https://deploy-preview-4319--kongdocs.netlify.app/gateway/latest/availability-stages/.

Reply:
+1